ios/cocoapods_processes.md

```diff
@@ -0,0 +1,66 @@
+# Cocoapods Processes
+
+We like to make reusable code and cocoapods is a great way to distribute that code. Individual projects may be making their own pods, but we also maintain the `Intrepid` pod within the entire iOS team. This guide aims to cover the necessary steps for creating your own pod and also maintaining one, like we must do with the `Intrepid` pod.
+
+The goal of this doc is not to copy all the info available at [cocoapods.org](https://cocoapods.org/). This is specifically oriented towards helpful advice specific to Intrepid processes.
+
+## Building Your Own Pod
+
+The first thing to do is just start a new xcode project that holds all of your source components. There will really be two parts of your project. There will be the actual pod components that you are building to be reusable by others -- let's keep these in a folder called `Source`. Then there will be the rest of your project that should really act like a demo of the pod.
+
+Your structure could end up looking something like this:
+
+![pod folder structure](./images/pod_folder_setup.png)
```

Comment:
Good point, and that's not a perfect example anyways, so I'll make a new sample project to demonstrate.